lib/features/recurring_donations/create/widgets/create_recurring_donation_bottom_sheet.dart

```diff
@@ -0,0 +1,373 @@
+                        hintText: locals.selectRecipient,
+                        contentPadding: const EdgeInsets.all(20),
+                        prefixIcon: state.recipient.orgName.isNotEmpty
+                            ? Icon(
+                                CollecGroupType.getIconByType(
+                                  state.recipient.type,
+                                ),
+                                color: AppTheme.givtBlue,
+                              )
+                            : null,
+                        errorStyle: const TextStyle(
+                          height: 0,
+                        ),
+                        focusedErrorBorder: const UnderlineInputBorder(
+                          borderRadius: BorderRadius.only(
+                            bottomLeft: Radius.circular(5),
+                            bottomRight: Radius.circular(5),
+                          ),
+                        ),
+                        focusedBorder: const UnderlineInputBorder(
+                          borderRadius: BorderRadius.only(
+                            bottomLeft: Radius.circular(5),
+                            bottomRight: Radius.circular(5),
+                          ),
+                          borderSide: BorderSide(
+                            color: AppTheme.givtLightGreen,
+                            width: 8,
+                          ),
+                        ),
+                        enabledBorder: UnderlineInputBorder(
+                          borderRadius: const BorderRadius.only(
+                            bottomLeft: Radius.circular(5),
+                            bottomRight: Radius.circular(5),
+                          ),
+                          borderSide: BorderSide(
+                            color: state.recipient.orgName.isNotEmpty
+                                ? AppTheme.givtLightGreen
+                                : Colors.transparent,
+                            width: 8,
+                          ),
+                        ),
+                      ),
+                      validator: (value) {
+                        if (value == null || value.isEmpty) {
+                          return '';
+                        }
+                        return null;
+                      },
+                    ),
+                  ),
+                ),
+                _buildTextRow(text: locals.setupRecurringGiftText3),
+                TextFormField(
+                  readOnly: true,
+                  controller: TextEditingController(
+                    text: DateFormat('dd MMM yyyy').format(state.startDate),
+                  ),
+                  onTap: () async {
+                    final fromDate = await showDatePicker(
+                      context: context,
+                      initialDate: state.startDate,
+                      firstDate: DateTime.now(),
+                      lastDate: DateTime.now().add(const Duration(days: 365)),
```

Comment:
what if we have a year with 366 days?

Reply:
Maybe you could create a new datetime with something like:
DateTime(date.year + 1, date.month, date.day);

Reply:
It's a DatePicker you can always choose a different date, even in 2035

Reply:
If the requirement is to show next year, we should do that ;) and not 'almost' a year